src/main/java/edu/umich/its/cpm/BoxAuthUser.java

```diff
@@ -0,0 +1,90 @@
+package edu.umich.its.cpm;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.sql.Timestamp;
+import java.util.UUID;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.SequenceGenerator;
+import javax.persistence.Id;
+import javax.persistence.GenerationType;
+
+import lombok.Getter;
+import lombok.Setter;
+
+import com.fasterxml.jackson.annotation.JsonRawValue;
+
+@Entity
+public class BoxAuthUser {
+
+	private static final Logger log = LoggerFactory.getLogger(BoxAuthUser.class);
+	
+	/**
+	 * Primary key field
+	 * the CoSign user id
+	 */
+	@Id
+	@Column(name = "USER_ID", columnDefinition = "VARCHAR(99) NOT NULL", table = "BOX_AUTH_USER")
+	@Getter
+	@Setter
+	private String userId;
+
+	/**
+	 * state 
+	 * An arbitrary string of your choosing that will be included in the response 
+	 * to your application. Anything that might be useful for your application can 
+	 * be included. Box roundtrips this information back to your application, 
+	 * and strongly recommends that you include an anti-forgery token, 
+	 * and confirm it in the response to prevent CSRF attacks to your users.
+	 * null for self-migrated (non-batch) site migration
+	 */
+	@Column(name = "STATE", columnDefinition = "VARCHAR(99)", table = "BOX_AUTH_USER")
+	@Getter
+	@Setter
+	private String state;
+
+	/**
+	 * The access_token is the actual string needed to make API requests. 
+	 * Each access_token is valid for 1 hour. 
```

Comment:
Would it help to keep track of the token expiration time, when it's known?  It may save some time using an existing token and receive the response that it's expired.  The code could go immediately to the process of getting a new one instead.